modules/storage-rocksdb/src/main/java/org/apache/ignite/internal/storage/rocksdb/PartitionDataHelper.java

```diff
@@ -138,14 +142,24 @@ void putRowId(ByteBuffer keyBuffer, RowId rowId) {
         assert rowId.partitionId() == partitionId : rowId;
         assert keyBuffer.order() == KEY_BYTE_ORDER;
 
-        keyBuffer.putLong(normalize(rowId.mostSignificantBits()));
-        keyBuffer.putLong(normalize(rowId.leastSignificantBits()));
+        putUuid(keyBuffer, rowId.uuid());
+    }
+
+    static void putUuid(ByteBuffer buffer, UUID uuid) {
```

Comment:
Maybe a "rowIdUuid", because there's also "normalize", which may or may not be necessary for general-purpose UUIDs